### PR TITLE
Disable line length control, add average value for united area requests

### PR DESCRIPTION
**Pull request by liu-hanli**

### CODE REVIEW

File: openelevationservice/server/api/querybuilder.py [hunk 1]
```diff
@@ -14,7 +14,7 @@
 log = get_logger(__name__)
 
 coord_precision = float(SETTINGS['coord_precision'])
-division_limit = 1 / float(SETTINGS['maximum_nodes'])
+# division_limit = 1 / float(SETTINGS['maximum_nodes'])
```
Comment (franchu7) [inline openelevationservice/server/api/querybuilder.py:17]: Delete the commented out code if it is not necessary in the future

File: openelevationservice/server/api/views.py [hunk 2]
```diff
@@ -100,10 +101,10 @@ def elevationline():
...
                                           f'Invalid format_in value "{format_in}"')
         
-    if len(list(geom.coords)) > SETTINGS['maximum_nodes']:
-        raise api_exceptions.InvalidUsage(status_code=400,
-                                          error_code=4003,
-                                          message='Maximum number of nodes exceeded.')
+    # if len(list(geom.coords)) > SETTINGS['maximum_nodes']:
+    #     raise api_exceptions.InvalidUsage(status_code=400,
+    #                                       error_code=4003,
+    #                                       message='Maximum number of nodes exceeded.')
```
Comment (franchu7) [inline openelevationservice/server/api/views.py:107]: Delete the commented out code if it is not necessary in the future

**Review by franchu7 — APPROVED** [review]
OK. Well done!

File: openelevationservice/server/api/querybuilder.py (hunk 1 above)
Comment (liu-hanli) [reply openelevationservice/server/api/querybuilder.py:17]: DONE

File: openelevationservice/server/api/views.py (hunk 2 above)
Comment (liu-hanli) [reply openelevationservice/server/api/views.py:107]: DONE

File: openelevationservice/server/api/querybuilder.py [hunk 3]
```diff
@@ -258,7 +259,7 @@ def line_elevation(geometry, format_out, dataset):
         if lineLen != 0:
             points_clause.insert(1, func.ST_LineInterpolatePoints(
                 geometry.wkt,
-                max(min(1, coord_precision / lineLen), division_limit)
+                min(1, coord_precision / lineLen) # max(min(1, coord_precision / lineLen), division_limit)
```
[the remark this answers is not in the document]
Comment (liu-hanli) [reply openelevationservice/server/api/querybuilder.py:262]: DONE

**Pull request merged**